elements/lisk-p2p/src/peer/base.ts

```diff
@@ -230,6 +263,10 @@ export class Peer extends EventEmitter {
 			};
 
 			if (message.event === REMOTE_EVENT_POST_NODE_INFO) {
+				this._peerStatusMessageRate += 1;
```

Comment:
this should not be incremented?

Reply:
oh I missed a separate counter for it and incrementing the config constant, I wonder how tests are passing 🤔 . I'll fix it